pm/queue.go

```diff
@@ -140,12 +148,14 @@ func (q *ticketQueue) startQueueLoop() {
 	}
 }
 
-// removeHead removes the head of the queue
-func (q *ticketQueue) removeHead() {
-	// Lock conditional variable while removing from the queue
-	q.cond.L.Lock()
-	q.queue[0] = nil
-	q.queue = q.queue[1:]
-	atomic.AddInt32(&q.queueLen, -1)
-	q.cond.L.Unlock()
+// pop returns the head of the queue
+func (q *ticketQueue) pop() (*SignedTicket, error) {
```

Comment:
This can be removed.

Reply:
Looks like a9d18c1 updates a test, but does not remove this test.

Reply:
I didn't save the file 🤦‍♂️ e90d987